### PR TITLE
🚧 Consistent dataloader for benchmark

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes
Consistent dataloader for benchmark
Transforms.RandomCrop/Clip will generate different tensors every time call dataloader creation function. It bring the inconsistent model accuracy measurement.

This PR is a workaround used to ensure the input data is consist for torch model and onnx model.

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

File: examples/resnet/user_script.py
Comment (jambayk) [inline examples/resnet/user_script.py:1]: What do you think about setting random seeds in the example's test script? https://discuss.pytorch.org/t/random-transforms-not-consistent-with-the-same-seed/63816

Comment (trajepl) [reply examples/resnet/user_script.py:1]: Thanks for offline syncing. Both of numpy/random/torch.random did not work on this case.

Comment (trajepl) [reply examples/resnet/user_script.py:1]: Let us disable random transforms for this case.

**Review by jambayk — APPROVED** [review]

**Review by jambayk — APPROVED** [review]

File: examples/resnet/user_script.py [hunk 1]
```diff
@@ -17,6 +20,14 @@
 # Common Dataset
 # -------------------------------------------------------------------------
 
+seed = 0
+
+np.random.seed(seed)
```
[the remark this answers is not in the document]
Comment (trajepl) [reply examples/resnet/user_script.py:25]: https://pytorch.org/docs/stable/notes/randomness.html
Based on this, better to have pinned seed value for ramdom/np.random/torch random all.
